tokens/web/deprecated-2019-fall.json5

```diff
@@ -0,0 +1,611 @@
+{
+  text: {
+
+    header: {
+
+      // header-1
+      '1': {
+        family: {
+          value: '{options.font.family.serif.value}',
```

Comment:
I just realized these are breaking changes for this, is that ok? 

Sure we're deprecating them, but this is also changing actual font families and stuff (like using Stuart instead of Roboto, etc.) so it isn't a backwards compatible deprecation...but are people going to be allowed to stay on the old font stack if they choose? How is that being handled?

Reply:
@mhewson This was the plan, correct? All tokens get the updated font stack. I think it still is backwards compatible because the token names wont change, only their values. Changing a value doesn't qualify as a breaking change